src/test/java/org/gridsuite/ds/server/controller/DynamicSimulationControllerIEEE14Test.java

```diff
@@ -232,9 +238,19 @@ public void test01GivenCurvesAndEvents() throws Exception {
         String jsonResultTimeSeries = TimeSeries.toJson(resultTimeSeries);
 
         // export result to file
-        FileUtils.writeStringToFile(this, outputDir + RESOURCE_PATH_DELIMITER + "exported_" + RESULT_SIM_JSON, jsonResultTimeSeries);
+        FileUtils.writeBytesToFile(this, outputDir + RESOURCE_PATH_DELIMITER + "exported_" + RESULT_SIM_JSON, jsonResultTimeSeries.getBytes());
 
         // compare result only timeseries
         assertThat(objectMapper.readTree(jsonResultTimeSeries)).isEqualTo(objectMapper.readTree(jsonExpectedTimeSeries));
+
+        // check dump file not empty
+        byte[] outputState = dynamicSimulationResultService.getOutputState(runUuid);
+        assertThat(outputState).isNotEmpty();
+        logger.info("Size of zipped output state = {} KB ", outputState.length / 1024);
```

Comment:
are you sure about the logging level ?
shouldn't be logging in debug level here ?

Reply:
NO ACTION